sagemaker-debugger/tensorflow2/tensorflow2_zero_code_change/src/keras_fit_zero_code_change.py

```diff
@@ -0,0 +1,71 @@
+"""
+This script is a ResNet training script which uses Tensorflow's keras fit() API.
+Using this script with the official SageMaker Framework container (i.e. AWS Deep Learning Container), enables you to make use of the SageMaker Debugger without any Debugger related
+additions to the training script.
+For more information, please refer to https://github.com/awslabs/sagemaker-debugger/blob/master/docs/sagemaker.md
```

Comment:
link about zero code change and what is saved by ZCC by default and what can be configured in estimator sdk(default collections , custom collections etc)

Reply:
done